packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -155,6 +155,9 @@ class StackViewport extends Viewport implements IStackViewport {
   public modality: string; // this is needed for tools
   public scaling: Scaling;
 
+  // Camera properties
+  private initialViewUp: Point3;
```

Comment:
I had to introduce this instance variable because initialCamera might still be pointing to the previous image's camera (e.g. in _updateActorToDisplayImageId).